client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -151,16 +190,37 @@ export const HoldList = ( { holds, isPlaceholder, translate }: Props ) => {
 							</div>
 						)
 					) }
-			</Card>
-		</div>
+			</div>
+		</>
 	);
 };
 
+function getCardHeading( context: string | null, translate: LocalizeProps[ 'translate' ] ) {
+	switch ( context ) {
+		case 'plugins':
+			return translate( "To install plugins, you'll need a couple things:" );
+		case 'themes':
+			return translate( "To install themes, you'll need a couple things:" );
+		default:
+			return translate( "To continue, you'll need a couple things:" );
```

Comment:
as above: missing `of`

Reply:
I was not typing well that day it seems! 😅 

The "couple things" came from the design. I don't mind it, but do you feel it's too informal compared to our usual voice?

Reply:
I've heard it before, and shudder every time :)

I know it's part of North-American English, but in my judgement, readers from the UK/AU/NZ/CA would find it unusual. Confusing things further, "couple things" is also used to describe activities that take place between couples in a relationship.

We're speaking to a wider audience, and therefore I'd feel comfortable overriding the copy writers on this one, if only to make the sentence clearer, and easier to read.

Just my 1.39c (after AUD->USD conversion)